templates/default/mgm-server-start.sh.erb

```diff
@@ -23,11 +23,20 @@ if [ ! -e <%= node['mysql']['base_dir'] %>/bin/ndb_mgmd ] ; then
     exit 3
 fi
 
+INITIAL_START_ARG=
+if [ -n "$INITIAL_START" ]; then
+    INITIAL_START_ARG="--initial"
```

Comment:
this will not work, during baking and on-premise setups it will fail since `INITIAL_START` is not set anywhere
I think the default value should be `--initial` ?

why do we actually care about that one for the mgm server? I thought that should be always set for mgmd?

Reply:
--initial is one way of setting parameters when starting ndb_mgmd, might be the only one we currently use, but I think Vincent is more up to date what we actually do with the MGM Server

Reply:
The ndb-agent should have the possibility to start the MGMd without `--initial`.

Reply:
I.e. don't just make `--initial` default.

Reply:
Our default behaviour up until now was to always run `--initial` on starting management service, can you elaborate on why this changes now and what would are the implications for on-premise installation when we start the management server without `--initial` would it work normally?

Reply:
discussed on slack, I'll add the environment files with the default values as what we have now to keep the same setup as before